doc/glossary.rst

```diff
@@ -114,6 +114,12 @@ If you wish to add a missing term, please `create a new issue`_ or
         `Family-wise error rate`_ is the probability of making one or more
         false discoveries, or type I errors when performing multiple
         hypotheses tests.
+    
+    FWHM
+        `FWHM`_ stands for "full width at half maximum". In a distribution, it
+        refers to the difference between the two values of the independent
+        variable at which the dependent variable is equal to half of its
+        maximum value.
```

Comment:
The above  explanation makes sense for 1D variables, while it is often used for 3D data.
More simply, this is the width of a filter, expressed as the diameter of the area on which the filter value is above half it maximal value.

Reply:
Thanks! This is indeed much better and general.